panther_lights/src/animation/animation.py

```diff
@@ -9,6 +11,8 @@ def __init__(self, animation_description: dict, num_led: int, controller_freq: f
         self._num_led = num_led
         self._current_cycle = 1
```

Comment:
```suggestion
        self._current_cycle = 0
```
Maybe iterate cycles from 0?